build.gradle

```diff
@@ -1,96 +1,37 @@
+
 plugins {
-	id 'org.springframework.boot' version '2.5.3'
-	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-	id 'org.asciidoctor.convert' version '1.5.8'
-	id 'java'
-	id 'jacoco'
```

Comment:
We shouldn't delete jacoco and org.asciidoctor.convert: the first we'll need for sure when we set up tests, the second is more debatable, but would be safer to keep it